experimenter/experimenter/targeting/constants.py

```diff
@@ -1944,6 +1944,28 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+SEARCH_ROLLOUT_1 = NimbusTargetingConfig(
+    name="Search Rollout 1",
+    slug="search_rollout_1",
+    description="Search Rollout 1 Namespace",
+    targeting="",
```

Comment:
This should probably be `true` because I can see this easily getting goofed up when we mash together targeting stuff (like sticky)

Reply:
There are tests that create a JEXL out of each advanced targeting and validate it against both the python and js JEXL runtimes so if this generates something invalid I'd expect those tests to fail.

Reply:
Ah also, empty targeting will not be included in the targeting string at all:

https://github.com/mozilla/experimenter/blob/main/experimenter/experimenter/experiments/models.py#L474-L475

but the slug will be included in the namespace:

https://github.com/mozilla/experimenter/blob/main/experimenter/experimenter/experiments/models.py#L797-L798

which I think is waht we want here